### PR TITLE
Bugfix/uniformisation fix

### PR DESCRIPTION
**Pull request by whispyy**
Review uniformisation changes following CX guidelines. See UU-445

### CODE REVIEW

File: client/app/vrack/vrack.html [hunk 1]
```diff
@@ -113,12 +114,12 @@ <h2 class="vrack__section-title" data-translate="vrack_description_title"></h2>
...
         </section>
 
-        <section class="vrack__introduction">
-            <h2 class="vrack__section-title" data-translate="vrack_introduction_title"></h2>
-            <p class="vrack__text" data-translate="vrack_introduction_description"></p>
-            <p class="vrack__text" data-ng-if="VrackCtrl.changeOwnerUrl"
-                                   data-translate="vrack_introduction_change_owner"
-                                   data-translate-values="{ uri: VrackCtrl.changeOwnerUrl }"></p>
+        <section>
+            <cui-page-content-title data-text=" 'vrack_introduction_title' | translate"></cui-page-content-title>
```
Comment (Mboulianne) [inline client/app/vrack/vrack.html]: extra white space after data-text="

File: client/app/vrack/vrack.html [hunk 2]
```diff
@@ -1,68 +1,69 @@
...
+
+    <div>
+        <cui-message-container data-messages="VrackCtrl.messages"></cui-message-container>
+    </div>
 
+    <div class="cui-page__content">
         <section>
             <div ng-if="VrackCtrl.description || VrackCtrl.descriptionEditing">
-                <h2 class="vrack__section-title" data-translate="vrack_description_title"></h2>
+                <cui-page-content-title data-text=" 'vrack_description_title' | translate"></cui-page-content-title>
```
Comment (Mboulianne) [inline client/app/vrack/vrack.html]: extra white space after data-text="

File: client/app/vrack/vrack.html (hunk 1 above)
Comment (whispyy) [reply client/app/vrack/vrack.html]: done

File: client/app/vrack/vrack.html (hunk 2 above)
Comment (whispyy) [reply client/app/vrack/vrack.html]: done

File: client/app/cloud/project/storage/storage-details/storage-details.controller.js [hunk 3]
```diff
@@ -58,6 +58,18 @@ angular.module("managerApp").controller("RA.storageDetailsCtrl", [
...
 
+                // handle messages
+        $scope.messages = [];
+
+        function refreshMessage () {
+            $scope.messages = $scope.messageHandler.getMessages();
+        }
+
+        function loadMessage () {
+            CloudMessage.unSubscribe("iaas.pci-project.storage.detailsß");
```
Comment (Netnix) [inline client/app/cloud/project/storage/storage-details/storage-details.controller.js]: s7 ß ? Comment tu tapes ça au clavier?

Comment (whispyy) [reply client/app/cloud/project/storage/storage-details/storage-details.controller.js]: Congrats ! You found Charlie !

**Pull request merged**